### PR TITLE
GPL-3.0 License

### PR DESCRIPTION
**Pull request by prem-k-r**
@XengShi @itz-rj-here please check

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,678 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (itz-rj-here) [inline LICENSE:659]: Bruh. No year no name no program name -_-

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,678 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (itz-rj-here) [inline LICENSE:638]: "    Material You NewTab: A browser extension that transforms the default 
    new tab into a sleek, Pixel-inspired interface with a clock, weather 
    info, and multi-search options."
    
Add this here.

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,678 @@
...
+    Copyright (C) <year>  <name of author>
```
Comment (itz-rj-here) [inline LICENSE:639]: Uhhh.

**Review by itz-rj-here — CHANGES_REQUESTED** [review]
Other things are okay but needs some changes

**Comment by itz-rj-here** [comment]
@prem-k-r

**Comment by prem-k-r** [comment]
I have checked other projects licences, they have left it as it is.

**Comment by itz-rj-here** [comment]
> I have checked other projects licences, they have left it as it is.

It is required to rewrite these.

**Comment by itz-rj-here** [comment]
Fix those or its no use :)

**Comment by itz-rj-here** [comment]
Or use a unlicense 🤣 just kidding

**Comment by prem-k-r** [comment]
Done
.. did from mobile 😅

**Comment by itz-rj-here** [comment]
> Done .. did from mobile 😅

Bruh seriously. 💀

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,678 @@
+          Material You NewTab Copyright (c) 2023-2024 XengShi
```
Comment (itz-rj-here) [inline LICENSE]: Can you just remove this. Adding this at the top breaks the format of the license 😐

**Comment by itz-rj-here** [comment]
image
You broke the license format or it would have showed GPL-3.0

**Comment by prem-k-r** [comment]
It's probably because it is in the branch not in default branch (main)
Let me check

**Comment by itz-rj-here** [comment]
> It's probably because it is in the branch not in default branch (main) Let me check

i don't think soo

**Comment by prem-k-r** [comment]
Now it's showing 
Screenshot_2024_1106_175505

Screenshot_2024_1106_175647

**Review by itz-rj-here — APPROVED** [review]
Great Good to go now

File: LICENSE (hunk 4 above)
Comment (prem-k-r) [reply LICENSE]: Old file, already removed

Comment (itz-rj-here) [reply LICENSE]: Great.

**Comment by XengShi** [comment]
Thankyou so much @prem-k-r @itz-rj-here

**Pull request merged**